common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -304,16 +304,20 @@ def get_html(self):
             if xblock_settings and 'YOUTUBE_API_KEY' in xblock_settings:
                 yt_api_key = xblock_settings['YOUTUBE_API_KEY']
 
+        poster = None
+        if edxval_api and self.edx_video_id:
+            poster = edxval_api.get_course_video_image_url(
+                course_id=self.runtime.course_id.for_branch(None),
+                edx_video_id=self.edx_video_id.strip()
```

Comment:
@muzaffaryousaf There is one other place where we are doing [edx_video_id.strip()](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/video_module/video_module.py#L511) but i think it is safe.

@Qubad786 Do you think is there any chance that self.edx_video_id can be set to None [here](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/video_module/video_module.py#L511)?